operator-k8s/src/operator.rs

```diff
@@ -56,6 +59,19 @@ impl Operator {
         Ok(())
     }
 
+    /// Generate crds
+    ///
+    /// # Errors
+    ///
+    /// Return `Err` when run failed
+    #[inline]
+    pub fn generate_crds(&self) -> Result<()> {
+        let path = format!("{}_{}.yaml",&Cluster::api_resource().group,&Cluster::api_resource().kind.to_lowercase());
+        let mut file = File::create(path).unwrap();
```

Comment:
Please process the returned result of `File::create` instead of unwarping them directly. Otherwise, it will lead to panic.